### DISCUSSION TITLE
Typescript webpack issue

### DISCUSSION THREAD

**Discussion by emretinaztepe** [gildas-lormeau/zip.js#336] [General]
Thanks a lot for the library. It is super helpful!

When I try to add it to my webpack, I receive the error below:

Any suggestions?

ERROR in ./node_modules/@zip.js/zip.js/lib/zip-fs.js 37:17
Module parse failed: Unexpected token (37:17)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| let baseURL;
| try {
>       baseURL = import.meta.url;
| } catch (error) {
|       // ignored
 @ ./node_modules/@zip.js/zip.js/index.js

**Comment by gildas-lormeau** [comment]
What version of webpack are you using?

**Comment by gildas-lormeau** [comment]
You're probably using an old version of webpack. If you're using webpack 4, you can fix this problem by using this loader: https://www.npmjs.com/package/@open-wc/webpack-import-meta-loader. I'm moving this issue in the Discussion tab since it's not a bug in zip.js.
